content/blog/2019/11/2019-11-21-macos-native-installer-deprecation.adoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Jenkins macOS native installer deprecation"
+tags:
+- announcement
+- deprecation
+- macos
+- platform-sig
+- platforms
+authors:
+- oleg_nenashev
+---
+
+In addition to WAR files and Docker images, the Jenkins project provides native installers for each weekly and LTS release. 
+There are installers available for Linux distributions, Windows, macOS and other operating systems.
+There are also installers provided by third parties.
+You can find the list of these installers on the link:/download[Downloads] page.
```

Comment:
Do we have a document for who is responsible to maintain those third parties? It might be useful if there's a link here.

Reply:
We don't. It would be great if you could create a follow-up ticket for it